src/runtime_src/core/pcie/tools/xbflash.qspi/pcidev.cpp

```diff
@@ -87,6 +88,9 @@ pci_device::
 pci_device(const std::string& sysfs, int ubar, size_t flash_off, std::string flash_type)
   : user_bar_index(ubar), flash_offset(flash_off), flash_type_str(flash_type)
 {
+	 
+  int pcmd,pcmd_new;
```

Comment:
pcmd_new is not required. One variable is enough.
the type should be uint32_t, or you can also define uint16_t or uint8_t, then read/write length will be 2 or 1.